### DISCUSSION TITLE
Error when trying to download vivado on Mac

### DISCUSSION THREAD

**Discussion by Sgore9** [ichi4096/vivado-on-silicon-mac#53] [Q&A]
I was trying to download Vivado and I had like 15 gigabytes of the program downloaded but then it stopped with this error:

/home/user/scripts/install_vivado.sh: line 35: exit1: command not found

The only thing that I know is that I did not 'remove the quarantine attribute from /Users/sethgore/Downloads/vivado-on-silicon-mac-main/scripts/xvcd/bin/xvcd manually'

I'm not sure if that is what the issue was... I also don't know how to remove the quarantine attribute from the file my terminal mentioned. Its Aldo telling me to run cleanup.sh but I don't know how to.

**Comment by Sgore9** [comment]
Downloading files (15.84 GB / 20.62 GB) 129 h and 4 m(s) left at 12 KB/sec. \ERROR - There was an error downloading file: https://xilinx-ax-dl.entitlenow.com/dl/ul/2024/05/23/R212316774/rdi_0130_2024.1_0522_2023.xz?hash=6ymArw33FGY-JT-d4GKCNg&expires=1725849270&filename=rdi_0130_2024.1_0522_2023.xz&sessionid=44a19d72-5b59-4fda-8a61-9db54b1cdfe4 error was: There is an error while downloading file: https://xilinx-ax-dl.entitlenow.com/dl/ul/2024/05/23/R212316774/rdi_0130_2024.1_0522_2023.xz?hash=6ymArw33FGY-JT-d4GKCNg&expires=1725849270&filename=rdi_0130_2024.1_0522_2023.xz&sessionid=44a19d72-5b59-4fda-8a61-9db54b1cdfe4 
Downloading files (15.85 GB / 20.62 GB) 36 h and 10 m(s) left at 46 KB/sec. |INFO  - Internet connection validated, can connect to internet. 
                                                                                Downloading files (15.86 GB / 20.62 GB) 41 minutes left at 2 MB/sec. -ERROR - Checksum failed, expected: c4aa1f9099706d65265d3316d7fe72c8 but was: 1c0b1ceefef41079e8e4f1f2c692d9b0 for file https://xilinx-ax-dl.entitlenow.com/dl/ul/2024/05/23/R212317464/rdi_0782_2024.1_0522_2023.xz?hash=c4uh31_xf1o8jYM2NBsXeQ&expires=1725849270&filename=rdi_0782_2024.1_0522_2023.xz&sessionid=44a19d72-5b59-4fda-8a61-9db54b1cdfe4 
ERROR - Checksum failed, expected: 924a551f92e3473b50ea5ff43705bfa7 but was: 2df433be71c5b4c42cd14bd082969e33 for file https://xilinx-ax-dl.entitlenow.com/dl/ul/2024/05/23/R212316962/rdi_0272_2024.1_0522_2023.xz?hash=joIaiKqfQCi9GlZRKrmfvg&expires=1725849270&filename=rdi_0272_2024.1_0522_2023.xz&sessionid=44a19d72-5b59-4fda-8a61-9db54b1cdfe4 
Downloading files (15.75 GB / 20.62 GB) 3 h and 15 m(s) left at 522 KB/sec. |/home/user/installer/xsetup: line 115:   163 Killed                  ${X_JAVA_HOME}/bin/java ${ARGS} -cp ${X_CLASS_PATH} com.xilinx.installer.api.InstallerLauncher "$@"


I also got this message

**Reply by ichi4096** [reply]
exit1 was a typo but not that important.
The quarantine attribute is automatically added when downloading the repo as a zip-file but not when cloning via git. I should add a test whether it exists at all before trying to remove it, but that error message is also irrelevant.

Sometimes the installer just crashes because the downloaded files are corrupted for some odd reason. Increasing the allocated ressources for the Docker container _might_ help but for me just attempting the installation again was enough. Don't forget to run `cleanup.sh` beforehand.

**Comment by Sgore9** [comment]
So basically I should just redownload the files and allocate more resources for the docker container? Perfect

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by ichi4096** [reply]
Yeah, that's what worked for me given these errors.

**Comment by ichi4096** [comment]
Has this worked for You?

**Comment by Sgore9** [comment]
No lol, I just used an old windows laptop.
________________________________

**Reply by ichi4096** [reply]
Great if that works. If You still decide to install it on Your mac later on, make sure to redownload my repo since there have been some changes to the scripts.
